extensions/cms/src/cmsResource/tree/cmsResourceTreeNode.ts

```diff
@@ -77,8 +85,11 @@ export class CmsResourceTreeNode extends CmsResourceTreeNodeBase {
 					}
 				}
 			}, (error) => {
-				let errorText = localize('cms.errors.expandCmsFail', 'The Central Management Server {0} could not be found or is offline', this.name);
-				this.appContext.apiWrapper.showErrorMessage(error ? error : errorText);
+				this.treeChangeHandler.notifyNodeChanged(undefined);
+				if (error) {
+					this.appContext.apiWrapper.showErrorMessage(error.message);
```

Comment:
Pretty sure this can never be undefined so you can just get rid of the check here and always throw